requirements.txt

```diff
@@ -12,14 +12,14 @@
 # scripts/convert-requirements-to-conda-yml.py as though it can only be found
 # on pip.
 
-GDAL>=3.1.2,<3.3.0
+GDAL>=3.1.2,!=3.3.0  # 3.3.0 had a bug that broke our windows builds: https://github.com/OSGeo/gdal/issues/3898
```

Comment:
That's cool, I hadn't seen environment specifiers before! It doesn't look like conda supports a similar thing, so unless we have a particular reason to allow `3.3.0` on mac, I'm inclined to leave it as-is and not have to modify the `environment.yml` script

Reply:
I don't have a specific reason not to disallow 3.3.0 on mac other than it seems unnecessary to prevent someone from using a package version that isn't known to be a problem on a platform.

Since there's already a helpful comment explaining the situation, let's not worry about environment specifiers here.